pkg/cover/html.go

```diff
@@ -508,22 +517,31 @@ func (rg *ReportGenerator) DoCSV(w io.Writer, progs []Prog, coverFilter map[uint
 		return err
 	}
 	var data [][]string
-	for fname, file := range files {
-		for _, function := range file.functions {
-			data = append(data, []string{
-				file.module,
-				fname,
-				function.name,
-				strconv.Itoa(function.covered),
-				strconv.Itoa(function.pcs),
-			})
+	for _, files1 := range files {
```

Comment:
Nit: use a more descriptive name than files1?

Reply:
ok, will rename to mfiles to stand for files for the module